src/httpserver.cpp

```diff
@@ -510,8 +510,8 @@ void HTTPEvent::trigger(struct timeval* tv)
     else
         evtimer_add(ev, tv); // trigger after timeval passed
 }
-HTTPRequest::HTTPRequest(struct evhttp_request* _req) : req(_req),
-                                                       replySent(false)
+HTTPRequest::HTTPRequest(struct evhttp_request* _req, bool _replySent) : req(_req),
+                                                                         replySent(_replySent)
```

Comment:
style-nit: Could add a newline after `:` to save some whitespace

Reply:
Fixed in a way that `clang-format` appreciates :)

Reply:
I like to put each member in a separate line so that adding or removing one is less likely to conflict with other related changes, but no biggie

Reply:
Can `.clang-format` be changed to get that behaviour?

Reply:
I already knows it, you just have to add the newlines manually:

```
$ git diff && (git diff|cfd) && git diff 
diff --git a/src/httpserver.cpp b/src/httpserver.cpp
index 7282b517f4..3b2e644319 100644
--- a/src/httpserver.cpp
+++ b/src/httpserver.cpp
@@ -510,7 +510,9 @@ void HTTPEvent::trigger(struct timeval* tv)
     else
         evtimer_add(ev, tv); // trigger after timeval passed
 }
-HTTPRequest::HTTPRequest(struct evhttp_request* _req, bool _replySent) : req(_req), replySent(_replySent)
+HTTPRequest::HTTPRequest(struct evhttp_request* _req, bool _replySent)
+                                                                       : req(_req),
+                                                                       replySent(_replySent)
 {
 }
 
Formatting src/httpserver.cpp
diff --git a/src/httpserver.cpp b/src/httpserver.cpp
index 7282b517f4..96a525bf86 100644
--- a/src/httpserver.cpp
+++ b/src/httpserver.cpp
@@ -510,7 +510,9 @@ void HTTPEvent::trigger(struct timeval* tv)
     else
         evtimer_add(ev, tv); // trigger after timeval passed
 }
-HTTPRequest::HTTPRequest(struct evhttp_request* _req, bool _replySent) : req(_req), replySent(_replySent)
+HTTPRequest::HTTPRequest(struct evhttp_request* _req, bool _replySent)
+    : req(_req),
+      replySent(_replySent)
 {
 }
 
```

```
$ which cfd
cfd ()
{ 
    ~/workspace/btc_bitcoin_core/contrib/devtools/clang-format-diff.py -p1 -i -v
}